src/Week6_CodingTasks/Task2_Mahbuba.java

```diff
@@ -0,0 +1,22 @@
+package Week6_CodingTasks;
+
+public class Task2_Mahbuba {
+    //Array -- Find Maximum
+    //Write a method that can find the maximum number from an int Array
+    public static int findMaxArray(int[] arr) {
+        int max = arr[0]; //initialize max,it will hold value of first array
+        for (int i = 0; i < arr.length - 1; i++) { //loop through array
```

Comment:
I think if you are putting i < arr.length - there should not be length-1, because if for example your length is 5, so it means indexes are from 0 to 4. And if you are putting only < (less than 5 in our case) sign then it will not use 5, it will stop at 4, so we are good. 
But if you put <= then it should be length-1;

Reply:
Oh, yes.Thank you for catching that!
I updated my method.

Reply:
For some reason I don't see updates.